typescript/packages/reality-data-transfer/package.json

```diff
@@ -2,19 +2,27 @@
 	"name": "@itwin/reality-data-transfer",
 	"version": "0.1.0-dev.1",
 	"description": "Utils to upload local data to context share",
-	"main": "lib/cjs/index.js",
-	"module": "lib/esm/index.js",
-	"typings": "lib/cjs/index",
+	"exports": {
+		"node": {
+		  "types": "./lib/cjs/NodeMain.d.ts",
+		  "import": "./lib/esm/NodeMain.js",
+		  "require": "./lib/cjs/NodeMain.js"
+		},
+		"default": {
+		  "types": "./lib/esm/BrowserMain.d.ts",
+		  "import": "./lib/esm/BrowserMain.js"
+		}
+	  },
 	"scripts": {
 		"build": "npm run -s build:cjs && npm run -s build:esm",
 		"build:cjs": "tsc 1>&2 --outDir lib/cjs",
-		"build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+		"build:esm": "tsc 1>&2 --module ESNext --outDir lib/esm",
```

Comment:
why the change here?

Reply:
you modified https://www.typescriptlang.org/tsconfig#module, https://www.typescriptlang.org/docs/handbook/modules/reference.html#the-module-compiler-option


i think its ok two leace it as one tsconfig file that you overrite inline in the scripts section. this is what we do in core

Reply:
i was asking why changing it from es2020 to esnext here

Reply:
I don't remember why, and doesn't seem to be very useful. I will revert this change.

Reply:
Done